src/main/java/uk/gov/hmcts/divorce/solicitor/event/Applicant2SolicitorSwitchToSoleCo.java

```diff
@@ -89,7 +89,7 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(CaseDetails<C
 
         switchToSoleService.switchApplicantData(data);
 
-        generateConditionalOrderAnswersDocument.apply(details);
+        generateConditionalOrderAnswersDocument.apply(details, data.getApplicant2().getLanguagePreference());
```

Comment:
Should this be using applicant 1 language preference here?  As the applicants have been switch on line 90 above.